docs/_docs/install.md

```diff
@@ -74,6 +74,7 @@ CMake Option                         | Description
 `-DENABLE_PYTHON=OFF`                | Build python bindings (experimental)
 `-DENABLE_FREESASA=ON`               | Enable SASA routines (external download)
 `-DENABLE_TBB=OFF`                   | Build with Intel Threading Building Blocks (experimental)
+`-DENABLE_PCG=OFF`                   | Use PCG random number generator instead of C++'s Mersenne Twister
```

Comment:
Maybe we should keep it on as default due to PCG superiority over Mersenne Twister?

Reply:
I agree it should be better but we currently don't know so much about it. I think we can leave it on by default after the next release.